conduct/enforcement.md

```diff
@@ -83,6 +84,28 @@ satisfactory resolution was achieved or not) and decide on any additional
 action deemed necessary.
 
 
+## Acting unilaterally
+
+In some cases, individuals may violate the CoC in online or in-person situations.
+If any community member deems the action to be harmful to the community,
+they may act immediately, before reaching consensus, to defuse the situation.
+In ongoing situations, any member may at their discretion employ any of the
+tools available in this enforcement manual, including bans and blocks online,
+or removal from a physical space.
```

Comment:
Since this paragraph is about unilateral actions by any community member to *defuse* a situation, can we reserve the more serious actions (like bans and blocks) that are targeted towards individuals to the CoC committee, and instead list here "cooling down" actions that are not targeted to any specific individual to defuse a situation, like locking a thread?

If someone needs to be physically removed, that implies an urgency that may fall under the physical safety category below, so I don't think it needs to be here.